apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1237,7 +1237,7 @@ else:
         $("#contentSummaryModal").modal("show");
         $.getJSON("${url('filebrowser:content_summary', path='')}" + encodeURIComponent(self.selectedFile().path), function (data) {
           if (data.status == 0) {
-            self.contentSummary(ko.mapping.fromJS(data.summary));
+            self.contentSummary(ko.mapping.fromJS(data));
```

Comment:
can you skip this mako change? as these are inline scrips changes it would affect my changes.